src/containers/FileEditorContainer.js

```diff
@@ -49,28 +52,28 @@ const FileEditorContainer = ({kfId, history, match}) => {
               },
             ]}
           >
-            {(updateFile, {_}) => {
-              return (
-                <FileEditor
-                  kfId={data.fileByKfId.kfId}
-                  name={fileNameInput}
-                  description={data.fileByKfId.description}
-                  fileType={fileType}
-                  selectFileType={selectFileType}
-                  onSubmit={e => onSubmit(e, updateFile)}
-                  onNameChange={e => setFileName(e.target.value)}
-                  onDescriptionChange={e => setFileDescription(e.target.value)}
-                />
-              );
-            }}
+            {(updateFile, {_}) =>
+              children
```

Comment:
Since you only want to render if there are children, you don't need the ternary and could just do `{children}`

Reply:
if `children` is null it throws an error though